intel_extension_for_transformers/llm/runtime/graph/models/gptj/gptj.cpp

```diff
@@ -72,10 +72,21 @@ static bool gptj_model_eval_internal(model_context& lctx, const model_token* tok
   const int n_embd = hparams.n_embd;
   const int n_layer = hparams.n_layer;
   const int n_ctx = hparams.n_ctx;
-  const int n_head = hparams.n_head;
+  int n_head = hparams.n_head;
+  const int head_size = n_embd / n_head;
   const int n_vocab = hparams.n_vocab;
   const int n_rot = hparams.n_rot;
 
+  bool enable_tp = false;
+#ifdef NE_TP_MODEL
```

Comment:
detect whether TP available automatically or pass the value by environment variables.

Reply:
so for a binary we released, we should always enable this option in cmake and it will only be executed when there are more than 1 node?

if so, can we remove this macro and set this true always in cmake?

Reply:
I don't like macros across all models which might be specific.